sdk/storage/azure-storage-blob-changefeed/azure/storage/blob/changefeed/_models.py

```diff
@@ -57,10 +57,13 @@ def __init__(
             extract_data=self._extract_data_cb,
             continuation_token=continuation_token or ""
         )
+        if continuation_token and container_client.primary_hostname != continuation_token['UrlHost']:
```

Comment:
make sure you validate cursor version! (idk if this is somewhere else already and I missed it)

Reply:
yeah its important to check in case someone gets a token from future version and uses it on this version